FeedStoreChallenge/Cache.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Cache.swift
+//  FeedStoreChallenge
+//
+//  Created by Hoff Henry Pereira da Silva on 27/07/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(Cache)
+final class Cache: NSManagedObject {
+	@NSManaged internal var timestamp: Date
+	@NSManaged internal var cachedFeedStoreImage: NSOrderedSet
```

Comment:
`internal` is a default access level, no need to define it explicitly. Please remove `internal` modifier
Also to follow up on my comment on `FeedStore.xcdatamodel`, I suggest to rename `cachedFeedStoreImage` to `images`

 ```suggestion
	@NSManaged var images: NSOrderedSet
```